index.html

```diff
@@ -69,6 +74,9 @@
             "IMSC1.1": {
               aliasOf: "ttml-imsc1.1-20181108"
             },
+            "IMSC1.2": {
+              aliasOf: "ttml-imsc1.2"
```

Comment:
Note for reviewers: since IMSC 1.2 Rec is not yet available, this seems to be the best link to use. We could consider switching it for the dated Rec link when that has been published, but it should follow automatically with a republication.

Reply:
I will now replace this with a ref to https://www.w3.org/TR/2020/REC-ttml-imsc1.2-20200804/

Reply:
Done in 4d88b3c.